packages/expo-updates/e2e/__tests__/setup/project.js

```diff
@@ -290,6 +273,58 @@ async function initAsync(projectRoot, { repoRoot, runtimeVersion, localCliBin })
     ],
     { cwd: projectRoot, stdio: 'inherit' }
   );
+}
+
+async function initAsync(
+  projectRoot,
+  {
+    repoRoot,
+    runtimeVersion,
+    localCliBin,
+    configureE2E = true,
+    transformAppJson = transformAppJsonForE2E,
+  }
+) {
+  console.log('Creating expo app');
+  const workingDir = path.dirname(projectRoot);
+  const projectName = path.basename(projectRoot);
+
+  // initialize project (do not do NPM install, we do that later)
+  await spawnAsync('yarn', ['create', 'expo-app', projectName, '--yes', '--no-install'], {
+    cwd: workingDir,
+    stdio: 'inherit',
+  });
+
+  if (configureE2E) {
+    await prepareLocalUpdatesModule(repoRoot);
+  }
+
+  await preparePackageJson(projectRoot, repoRoot, configureE2E);
+
+  // Now we do NPM install
+  await spawnAsync('yarn', [], {
+    cwd: projectRoot,
+    stdio: 'inherit',
+  });
+
+  /*
+  // Add dependencies from tgz files
```

Comment:
this seems like it's just doing a `yarn add`? is this tgz since it's in the same workspace?

Reply:
Probably should either remove this commented-out code or clarify why it's commented out in the comment.

Reply:
Removing